active/tab/tabcontrol.md

```diff
@@ -0,0 +1,580 @@
+            <TabViewItem Icon="Document" Header="Document 3" />
+        </TabView>
+
+        <Grid x:Name="CustomDragRegion" Width="200" Height="40" HorizontalAlignment="Right" VerticalAlignment="Top" />
+    </Grid>
+</Page>
+```
+
+``` csharp
+// Customize the titlebar area using the guidance from here: https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar
+public MainPage()
+{
+    this.InitializeComponent();
+
+    var coreTitleBar = CoreApplication.GetCurrentView().TitleBar;
+    coreTitleBar.ExtendViewIntoTitleBar = true;
+
+    Window.Current.SetTitleBar(CustomDragRegion);
+}
+```
+
+## Create a new window when tearing out tabs
+
+![An example picture showing that TabView can be torn out and moved to its own window.](images/tab-new-window.png)
+
+See the [TabView tear out sample](https://github.com/windows-toolkit/Sample-TabView-TearOff/tree/master/TabViewTear) for a more complete sample.
+
+``` xml
+<TabView CanDragTabs="True"
+            CanReorderTabs="True"
+            TabDraggedOutside="TabView_TabDraggedOutside">
+```
+``` csharp
+// NOTE: The app is responsible for writing this code. We will provide a sample that may look something like:
+private async void TabView_TabDraggedOutside(object sender, TabDraggedOutsideEventArgs e)
+{
+    // Create a new AppWindow
+    AppWindow newWindow = await AppWindow.TryCreateAsync();
+
+    // Create the content for the new window
+    var newPage = new MainPage();
+
+    // Remove tab from existing list
+    Tabs.TabItems.Remove(e.Tab);
+
+    // Add tab to list of Tabs on new page
+    newPage.AddItemToTabs(e.Tab);
+
+    // Set the Window's content to the new page
+    ElementCompositionPreview.SetAppWindowContent(newWindow, newPage);
+
+    // Show the window
+    await newWindow.TryShowAsync();
+}
+```
+
+## Databind to a set of tabs 
+
+``` xml
+<TabView TabItemsSource="{x:Bind TabViewItemCollection}" />
+```
+
+## Add a new tab using the Add (+) Tab Button
+The TabView control provides an Add (+) Tab button at the end of the tab strip. When the user clicks the Add Tab button, the control will fire an event which you can listen to in order to add a new tab.
+
+``` xml
+<controls:TabView
+    x:Name="TabRoot"
+    AddTabButtonClick="AddTabButton_Click"
+    >
+    <!-- ... some tabs ... -->
+</controls:TabView>
+```
+
+``` csharp
+
+private void AddTabButton_Click(TabView sender, TabViewAddTabButtonClickEventArgs e)
+{
+    CreateNewTab();
+}
+
+private void CreateNewTab()
+{
+    // MyDocument is a data object that represents the content inside a Tab
+    MyDocument doc = new MyDocument();
+    doc.Title = "New Document";
+    doc.Content = GenerateBaconIpsum();
+    doc.Icon = new SymbolIcon(Symbol.Document);
+
+    // Create a new TabViewItem from the MyDocument object and add it to the TabItems collection
+    TabRoot.TabItems.Add(CreateNewTabFromDocument(doc));
+}
+
+private TabViewItem CreateNewTabFromDocument(MyDocument doc)
+{
```

Comment:
Somewhat unrealistic in that there is nothing that lets us get from the `MyDocument` to the `TabViewItem` or vice versa. Should we do a `newItem.Tag = doc;`?

Reply:
Clarified the sample.